collect_app/src/main/java/org/odk/collect/android/activities/InstanceChooserList.java

```diff
@@ -264,8 +264,6 @@ private void logFormEdit(Cursor cursor) {
 
         if (status.equals(Instance.STATUS_INCOMPLETE)) {
             AnalyticsUtils.logFormEvent(AnalyticsEvents.EDIT_NON_FINALIZED_FORM, formId, formTitle);
-        } else if (status.equals(Instance.STATUS_COMPLETE)) {
-            AnalyticsUtils.logFormEvent(AnalyticsEvents.EDIT_FINALIZED_FORM, formId, formTitle);
```

Comment:
You're no longer able to edit finalised forms from Drafts so this would never fire!